### PR TITLE
Implement Multi-Step Create Wallet + Flow Type

### PR DESCRIPTION
**Pull request by kylanhurt**
The purpose of this task is to implement a multi-step wallet creation process in order to simplify the wallet creation process *and prevent the keyboard from covering up the dropdown lists*.

Asana Task: https://app.asana.com/0/361770107085503/426615385139838/f

### CODE REVIEW

**Comment by paullinator** [comment]
This PR breaks the build per travis. please fix

File: src/modules/UI/scenes/CreateWallet/CreateWalletConnector.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import CreateWallet from './CreateWallet.ui.js'
+
+export default CreateWallet
```
Comment (paullinator) [inline src/modules/UI/scenes/CreateWallet/CreateWalletConnector.js]: Do not use default exports. Per our convo in engineering meeting, use explicitly named exports in all new code

File: src/modules/Main.ui.js [hunk 2]
```diff
@@ -50,7 +50,10 @@ import SendConfirmation from './UI/scenes/SendConfirmation/index'
 import Scan from './UI/scenes/Scan/ScanConnector'
 import ExchangeConnector from '../connectors/scene/CryptoExchangeSceneConnector'
 import WalletList from './UI/scenes/WalletList/WalletListConnector'
-import CreateWallet from './UI/scenes/CreateWallet/CreateWalletConnector.js'
+import CreateWalletName from './UI/scenes/CreateWallet/CreateWalletConnector'
```
Comment (paullinator) [inline src/modules/Main.ui.js]: Do not use default exports. This applies to all imports added in this file. Leave any old code alone. No need to clean up right now, but do make sure new code doesn't use default exports.

**Comment by paullinator** [comment]
Still getting unresolved module on running app

**Comment by paullinator** [comment]
```
Bundling `index.ios.js`  [development, non-minified, hmr disabled]  6.4% (535/2114), failed.
error: bundling failed: "Unable to resolve module `./UI/scenes/CreateWallet/CreateWalletConnector` from `/Users/paul/git/edge-react-gui/src/modules/Main.ui.js`: could not resolve `/Users/paul/git/edge-react-gui/src/modules/UI/scenes/CreateWallet/CreateWalletConnector' as a folder: it did not contain a package, nor an index file"```

**Comment by paullinator** [comment]
Note that I am testing this by doing a local merge from `createWalletMerge` to `develop` and building and running it from there.

**Comment by kylanhurt** [comment]
FYI I am closing this PR and opening a new one with a fresh branch:
https://github.com/Airbitz/edge-react-gui/pull/326

**Pull request closed without merging**